scalding-hraven/src/main/scala/com/twitter/scalding/hraven/reducer_estimation/HRavenHistoryService.scala

```diff
@@ -38,6 +38,15 @@ object HRavenHistoryService extends HistoryService {
 
   private val LOG = LoggerFactory.getLogger(this.getClass)
 
+  // List of fields that we consume from fetchTaskDetails api.
+  // This is sent to hraven service to filter the response data
+  // and avoid hitting http content length limit on hraven side.
+  private val TaskDetailFields = List(
```

Comment:
We could make this overridable as Option[List[String]] so any external cases still work with the older hraven version.